.buildbot.sh

```diff
@@ -48,7 +48,9 @@ args=(
     --ssh-port $SSHPORT
     --ssh-key $HOME/.ssh/id_ed25519.pub
 )
-BUILDBOT_PLATFORM=riscv64-purecap python3 tests/run_cheri_examples.py "${args[@]}"
+export BUILDBOT_PLATFORM=riscv64-purecap
+args=${args[@]}
+script -c "python3 tests/run_cheri_examples.py $args"
```

Comment:
This will create a file `typescript` (no, not the modern TypeScript :)). Should we `script -c "python3 tests/run_cheri_examples.py $args" > /dev/null` to avoid that?

Reply:
Good point; I had missed that. Fixed in https://github.com/capablevms/cheri-examples/pull/93/commits/d43d04939529b072c7eeec7be1e95017690303d6 (just redirecting was not sufficient from my quick tests).